### ISSUE TITLE
'VBN' object has no attribute 'dtype'

### ISSUE THREAD

**Issue by hadaev8** [tensorflow/tensorflow#20431]
Its broken or I do wrong something?

version 1.9.0-rc1

i = tf.layers.conv2d_transpose(i, filters, 4, strides=2, activation=tf.nn.leaky_relu, padding='same')
i = tf.contrib.gan.features.VBN(i)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by hadaev8** [comment]
Its google colab virtual machine

**Comment by yongtang** [comment]
@hadaev8 I am not able to reproduce with v1.9.0. Can you share the full code snippet that trigger the error?

**Comment by joel-shor** [comment]
I'm closing this out as not reproducible unless someone objects.

**Comment by hadaev8** [comment]
https://pastebin.com/czeQ3UUv
Here is my code, checked on colab tensorflow 1.9.0 rc2 just now
